Jobsway2goMvc/Models/Collection.cs

```diff
@@ -12,7 +12,7 @@ public class Collection
         public int Id { get; set; }
         public ApplicationUser User { get; set; }
         public string Name { get; set; }
-        //public List<Job> Job { get; set; }
+        public List<Job> Job { get; set; }
         //public List<Post> Post { get; set; }
```

Comment:
Rename `Job` to `Jobs` and make sure to add migration and update database if needed